Gemfile.lock

```diff
@@ -0,0 +1,106 @@
+PATH
```

Comment:
@caitmich why is this file here?

Reply:
This was mentioned here: https://github.com/dradis/dradis-pentera/pull/1#discussion_r1483194402. I decided to keep this file because of this reasoning:

> Ideally, Gemfile.lock is among the tracked files so the bundle environment will be similar among devs for situations like running the specs and building the gem.

Note that the specs here can be executed locally (no need for the Dradis base) so the bundle context will matter.

Reply:
My understanding is that .lock shouldn't be part of gem repos, and it's never been. If we need to change directions, it has to be across the board, not in a 1-off repo.